src/components/MerchantsPage/styles.module.scss

```diff
@@ -152,6 +152,39 @@
   }
 }
 
+.bannerContentContainer {
+  // grid relative to parent container
+  grid-row: 1;
+  grid-column: 1;
+  z-index: 1;
+
+  // creating grid container for its own children components
+  display: grid;
```

Comment:
Why not use flexbox?

Reply:
I 100% copied this from the NYC Map banner implementation. If I did it again from scratch, I would definitely consider flexbox.